framework/sandstone.cpp

```diff
@@ -2611,70 +2610,93 @@ static int exec_mode_run(int argc, char **argv)
 // Triage run attempts to figure out which socket(s) are causing test failures.
 // It simply removes sockets from the cpu_set_t one by one until the failures no
 // longer observed.
```

Comment:
@busykai: why are we doing negative testing instead of positive? That is, why do we test all-but-one to find out when it stops failing, instead of testing each socket individually?

Reply:
If I recall correctly, we thought that it's incorrect to assume that a test will fail if scoped to faulty socket only.

Reply:
Okay, so let's not change behaviour now.

But we may before too long once I'm done with the socket and sub-socket grouping work.